routes/users.js

```diff
@@ -87,27 +198,54 @@ router.post("/create", Utils.ensureAuthenticated, async (req, res, next) => {
         res.json({ message: "User created successfully" });
       });
     } else {
-      res.status(401).json({ message: "You are not authorized to do this." });
+      res.status(403).json({ message: "You are not authorized to do this." });
     }
   } catch (err) {
     console.log(err);
     res.status(500).json({ message: err.toString() });
   }
 });
 
-/** PUT - Route serving to update a user admin privilege in the application.
- * @name router.put('/update')
- * @function
- * @memberof module:routes/users
- * @param {number} req.body[0].steam_id - Steam ID of the user being edited.
- * @param {number} [req.body[0].admin] - Integer determining if a user is an admin of the system. Either 1 or 0.
- * @param {number} [req.body[0].super_admin] - Integer determining if a user is a super admin of the system. Either 1 or 0.
- * @param {String} [req.body[0].name] - Display name of the user being updated.
- * @param {string} [req.body[0].small_image] - Akamai Steam URL to the small profile image.
- * @param {string} [req.body[0].medium_image] - Akamai Steam URL to the medium profile image.
- * @param {string} [req.body[0].large_image] - Akamai Steam URL to the large profile image.
+
+
+ /**
+ * @swagger
+ *
+ * /users:
+ *   put:
+ *     description: Create user
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      description: Optional description in *Markdown*
+ *      required: true
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/NewUser'
+ *     tags:
+ *       - users
+ *     responses:
+ *       200:
+ *         description: User
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               schema:
+ *                 $ref: '#/components/schemas/User'
+ *       403:
+ *         $ref: '#/components/responses/Unauthorized'
+ *       412:
+ *         description: Nothing to update
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/SimpleResponse'
+ *       500:
+ *         $ref: '#/components/responses/Error'
  */
-router.put("/update", Utils.ensureAuthenticated, async (req, res, next) => {
+router.put("/", Utils.ensureAuthenticated, async (req, res, next) => {
```

Comment:
As I've stated in the PR, I'm not opposed to this, this was me just importing what was directly from get5-web. I'll need to sleep on it for now, but don't remove these just yet! And once again on a comment, just gotta update test cases. I think I can even give it a go and update them as needed as well.